src/commands/client/info/index.ts

```diff
@@ -8,8 +9,18 @@ export type InfoResult = {
   similarityFunction: "COSINE" | "EUCLIDEAN" | "DOT_PRODUCT";
 };
 
+type InfoEndpointVariants = `info` | `info/${NAMESPACE}`;
```

Comment:
A note: Info is not implemented per-namespace, so it should always be "/info". Also, it has a new field called `namespaces`

```kotlin
data class NamespaceInfo(val vectorCount: Long, val pendingVectorCount: Int)

data class IndexInfo(val vectorCount: Long, val pendingVectorCount: Int, val indexSize: Long, val dimension: Int, val similarityFunction: String, val namespaces: Map<String, NamespaceInfo>
```